src/ChildProcessUtilities.js

```diff
@@ -33,7 +36,8 @@ export default class ChildProcessUtilities {
 
   static execSync(command, opts) {
     const mergedOpts = objectAssign({
-      encoding: "utf8"
+      encoding: "utf8",
```

Comment:
Huh... yeah, looks like it's been the default since at least [0.10](https://nodejs.org/docs/v0.10.0/api/child_process.html#child_process_child_process_exec_command_options_callback).